.github/workflows/test.yml

```diff
@@ -68,8 +74,20 @@ jobs:
           vim --version
           make --keep-going $MAKE_TARGET
 
+      - name: Commit and push
+        if: success() && matrix.make_target == 'fast-test' && github.ref_name == 'master'
+        run: |
+          if ! make fast-check; then
+            git config --local user.name 'github-actions[bot]'
+            git config --local user.email 'github-actions[bot]@users.noreply.github.com'
+            git remote set-url origin 'https://github-actions:${{ github.token }}@github.com/${{ github.repository }}'
+            git add js/vimlparser.js py/vimlparser.py
+            git commit -m 'generate js and py'
+            git push origin '${{ github.ref_name }}'
+          fi
+
       - name: Coverage from Python
-        if: matrix.make_target == 'test' && success()
+        if: success() && matrix.make_target == 'fast-test'
```

Comment:
This could run also in case of failure, no?  (i.e. in case of failed tests (or when "Commit and push") would fail) there could also be coverage to upload)
(IIRC `success() &&` is implied, but `always() &&` could be used here)

(only minor, but noticed it when reviewing this briefly)

Reply:
Oh, thank you, I'll fix it. (why did I add `success()`..)